_guide/social-media.md

```diff
@@ -1,82 +1,47 @@
 ---
 layout: guide
-title: Social media
-description: Basic information on social media and accessibility.
-excerpt: Basic information on social media and accessibility.
+title: Social Media
+description: How to make your social media content accessible.
+excerpt:
 sidenav: docs
 categories:
   - Accessibility
 roles:
   - Content designer
 ---
 
-Basic information on social media and accessibility.
-
-## Tips
-
-### Text
-
-*   Use [plain language](https://accessibility.civicactions.com/guide/plain-language)
-*   Don't use all-caps
-*   Include descriptive text before links (example: "Read the help page for more info: [LINK]"
-
-
-### Hashtags
-
-*   Use camel case for hashtags (example: #WebAccessibility)
-
-
-### Images
-
-*   Avoid images of text
-*   Add alt text to all images
-*   Use static images instead of animated GIFs
-*   Make sure images have strong color contrast
-
-
-### Video
-
-*   Add captions and transcripts
-*   Review any auto-generated text
-
-
-### Emojis
-
-*   Use words instead of emojis to describe things
-*   Do not overuse emojis
-*   Add emojis at the end of messages
-
-
-## Hashtags
-
-Remember that it is easier to read and for assistive technolgy to pronounce if hashtags are written in #CamelCaseLetters. Each unique word in a hashtag should start with a capital letter. 
-
-Common accessibility-related hashtags:
-
-*   #accessibility
-*   #a11y
-*   #axschat
-*   #GAAD (during [Global Accessibility Awareness Day](https://globalaccessibilityawarenessday.org/))
-
+Social media is used to create and share ideas, information, and interests. Accessible social media content allows people with disabilities to engage with these virtual communities instead of being shut out of the conversation. Taking steps to make your social media content accessible means that more people will be able to engage with it!
+
+## Basic guidelines for social media
+
+1. Keep text as clear and simple as possible by using [plain language](https://accessibility.civicactions.com/guide/plain-language). 
+     * Use short sentences and choose words your audience uses. 
+     * Don't use all-caps. 
+     * Spell out abbreviations or acronyms. 
+     * Include descriptive text before links. For example, "Read the help page for more info: [LINK]"
```

Comment:
Can you use something other than square brackets for `[LINK]` or maybe decorate it with a code block like using backticks e.g.

```
`[LINK]`
```